packages/agent/src/invitation/handler.ts

```diff
@@ -251,7 +255,17 @@ export const receiveCredentialFromOpenId4VciOfferAuthenticatedChannel = async ({
     },
   })) as unknown as Array<string>
 
-  return credentials[0]
+  const credential = credentials[0]
+  // FIXME: not sure if the index of credentials will match?
```

Comment:
did it?

Reply:
😍

Reply:
we only request one, so not really needed to fix now